mm2src/mm2_main/src/ordermatch_tests.rs

```diff
@@ -1662,14 +1662,10 @@ fn pubkey_and_secret_for_test(passphrase: &str) -> (String, [u8; 32]) {
     (pubkey, secret)
 }
 
-fn p2p_context_mock() -> (mpsc::Sender<AdexBehaviourCmd>, mpsc::Receiver<AdexBehaviourCmd>) {
+fn p2p_context_mock(ctx: &MmArc) -> (mpsc::Sender<AdexBehaviourCmd>, mpsc::Receiver<AdexBehaviourCmd>) {
```

Comment:
Since there is no mocking happening anymore, maybe we can rename this function to `p2p_context_for_tests` and remove this attribute
https://github.com/KomodoPlatform/komodo-defi-framework/blob/a2cb99bfff328da74f9fb5cdfc877ffa1ef7bafe/mm2src/mm2_net/src/p2p.rs#L20

Reply:
I didn't want to do it right away, but I guess we can do that